_data/members.yaml

```
@@ -313,6 +313,28 @@ yt:
     multi-resolution data from astrophysical simulations,
     radio telescopes, and a burgeoning interdisciplinary community.
 
+heacit:
+  name: HEACIT
+  logo: CasA.png
+  url: https://heasarc.gsfc.nasa.gov/docs/heacit/
+  repositories:
+    somethingelse: https://github.com/HEACIT
+  mailinglist:
+    members: heacit@cfa.harvard.edu
```

Not sure how that needs to be formatted, but the HTML formatting apparently does not work with an email address here. Maybe try this:
```suggestion
    members: "mailto:heacit@cfa.harvard.edu"
```